### PR TITLE
Update set-up-snowflake-native-app.md

### PR DESCRIPTION
**Pull request by amychen1776**
## What are you changing in this pull request and why?
Updating the requirements page for the app

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ The following are the prerequisites for dbt Cloud and Snowflake.
 
 - You have **ACCOUNTADMIN** access in Snowflake.
 - Your Snowflake account must have access to the Native App/SPCS integration (PrPr until Summit) and NA/SPCS configurations (PuPr at end of June). If you're unsure, please check with your Snowflake account manager.
-- The Snowflake account must be in an AWS Region or Azure region. 
+- The Snowflake account must be in an AWS Region or Azure region.
+- You have access to Snowflake Cortex via your Snowflake permissions as well as the [region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). If you do not, dbt Ask will be broken.
```
Comment (amychen1776) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
- You have access to Snowflake Cortex via your Snowflake permissions as well as the [region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). If you do not, `Ask dbt` will be broken. 
```

Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and the [correct region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```

File: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ The following are the prerequisites for dbt Cloud and Snowflake.
 
 - You have **ACCOUNTADMIN** access in Snowflake.
 - Your Snowflake account must have access to the Native App/SPCS integration (PrPr until Summit) and NA/SPCS configurations (PuPr at end of June). If you're unsure, please check with your Snowflake account manager.
-- The Snowflake account must be in an AWS Region or Azure region. 
+- The Snowflake account must be in an AWS Region or Azure region.
+- You have access to Snowflake Cortex through your Snowflake permissions and the [correct region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and the [correct Snowflake Cortex region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```

**Review by mirnawong1 — APPROVED** [review]
lgtm and slight tweak on the below @amychen1776 ! please let me know if you're ok with this or have other thoughts! thank you so much!

> - You have access to Snowflake Cortex through your Snowflake permissions and the [correct Snowflake Cortex region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.

File: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md [hunk 3]
```diff
@@ -37,7 +37,8 @@ The following are the prerequisites for dbt Cloud and Snowflake.
 
 - You have **ACCOUNTADMIN** access in Snowflake.
 - Your Snowflake account must have access to the Native App/SPCS integration (PrPr until Summit) and NA/SPCS configurations (PuPr at end of June). If you're unsure, please check with your Snowflake account manager.
-- The Snowflake account must be in an AWS Region or Azure region. 
+- The Snowflake account must be in an AWS Region or Azure region.
+- You have access to Snowflake Cortex through your Snowflake permissions and the [correct Snowflake Cortex region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```
Comment (amychen1776) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and [Snowflake Cortex is available in your region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, Ask dbt will not work.
```

**Pull request merged**